jobs/aadAppRegistration.yml

```diff
@@ -92,10 +112,11 @@ jobs:
       -AADAppReplyUrls "${{ parameters.aadAppReplyUrls }}"
     displayName: 'Set AAD App Replay URLs'
 
+# Created new DNS records for websites
   - task: AzurePowerShell@3
     displayName: 'Set DNS records for web app'
     inputs:
-      azureSubscription: 'DCD-CFT-VH-Pilot'
+      azureSubscription: ${{ parameters.productionAzureSubscription }}
```

Comment:
i guess this is only for production, or can this parameter contain both production and non-production values?

Reply:
This is for making DNS changes. The DNS Zone is only in prod subscription.